### DISCUSSION TITLE
Docs are scarce about Self-hosted vs repo config

### DISCUSSION THREAD

**Discussion by bradydowling** [renovatebot/renovate#12675] [Suggest an Idea]
After setting up the renovate runner (in GitLab) I wasn't able to figure out why I kept getting `Repository is disabled - skipping` for the repo I cared about (and had explicitly included in my [self-hosted config file](https://docs.renovatebot.com/self-hosted-configuration/#repositories). [The GitLab docs](https://gitlab.com/renovate-bot/renovate-runner) (which may be owned by someone separate) don't mention anything about using a [repo config](https://docs.renovatebot.com/configuration-options/) so it seems like this is not needed.

Additionally, in [the main docs](https://docs.renovatebot.com/), there is nothing mentioned about the need for both a self-hosted config _and_ a repo config. The fact that they're separate sections on the same level seems to denote, at least to a degree, that they are an either/or deal.

![Screen Shot 2021-11-15 at 2 45 55 PM](https://user-images.githubusercontent.com/6668371/141844335-e00c16fc-4343-4b0f-aa57-498b3afbabff.png)

Did I miss something about this relationship somewhere or could this be added or clarified?

**Comment by bradydowling** [comment]
I'm happy to make a PR for this, just don't want to dive in without knowing it'll be looked at and strongly considered.

**Reply by viceice** [reply]
Repo config is normally always required, but can be disabled via global config.

https://docs.renovatebot.com/self-hosted-configuration/#requireconfig

**Comment by rarkins** [comment]
Should we add an "Overview" page to the Configuration which describes this?

**Reply by HonkingGoose** [reply]
That could help to clarify the situation a bit more.

Both the self-hosted config and the repository config pages do have a section on top that explains the scope of their content, but that's easy to miss if you're only looking at the sidebar.

> ## Self-Hosted configuration options [^self-hosted]
>
> The configuration options listed in this document are applicable to self-hosted instances of Renovate ("the bot").

> ## Configuration Options [^repo]
>
> This document describes all the configuration options you may configure in a Renovate configuration file. Any config you define applies to the whole repository (e.g. if you have a monorepo).

[^self-hosted]: https://docs.renovatebot.com/self-hosted-configuration/
[^repo]: https://docs.renovatebot.com/configuration-options/
